woocommerce-transbank/webpay.php

```diff
@@ -898,7 +917,14 @@ function pay_content($order_id) {
                     '<tr>' .
                     '<th scope="row">Tipo de Pago:</th>' .
                     '<td><span class="TP">' .
-                    $paymenCodeResult .
+                    $paymentType .
+                    '</span></td>' .
+                    '</tr>' .
+                    '<tr>' .
+                    '<tr>' .
```

Comment:
Is this <tr> being closed? I can't see the </tr>. Please check :)

Reply:
I've just pushed a fix for that